package.json

```diff
@@ -35,11 +35,24 @@
   "main": "./out/extension.js",
   "l10n": "./l10n",
   "contributes": {
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "salesforcedx-vscode-offline-app.configureLintingTools",
+          "when": "sfdx_project_opened"
```

Comment:
Oh cool, I didn't know about this functionality. That's a clean way to specify a precondition here.